index.html

```diff
@@ -373,6 +377,12 @@ <h3 class="section-title">Photos</h3>
     <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.min.js" integrity="sha384-+YQ4JLhjyBLPDQt//I+STsc9iw4uQqACwlvpslubQzn4u2UU2UFM80nGisd026JF" crossorigin="anonymous"></script>
     <script defer src="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/owl.carousel.min.js"></script>
     <script defer src="assets/js/main.js"></script>
+
+   
+    <a id="button"></a>
+
+    
+
```

Comment:
Whitspaces can be removed and can you change the id to something related to its function (for eg `id="go-to-top"`). Also  then move the anchor tag to somewhere near top of the `<body>`